package.json

```diff
@@ -39,17 +39,17 @@
     "node": ">=6.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.1.5",
-    "@babel/core": "^7.1.6",
-    "@babel/preset-env": "^7.1.6",
-    "eslint": "^5.9.0",
-    "eslint-config-metarhia": "^6.0.0",
-    "eslint-config-prettier": "^3.3.0",
-    "eslint-plugin-import": "^2.14.0",
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
```

Comment:
The latest version of `@babel/core` is `7.3.3`: https://www.npmjs.com/package/@babel/core